app/services/albums.js

```diff
@@ -1,28 +1,26 @@
-const request = require('request-promise');
-const errors = require('errors.js');
-const logger = require('/logger/index.js');
+const request = require('axios');
+const logger = require('/logger');
+const errors = require('../errors');
+const config = require('./config');
 
 const options = {
-  uri: process.env.ALBUMS_API_URL,
+  uri: config.common.api.albumsApiUrl,
   headers: { 'User-Agent': 'Request-Promise' },
   json: true
 };
 
 const get = data => {
   logger.info(`GET ${data}`);
-  request(data).catch(error => {
-    errors.defaultError(error);
-  });
+  return request
+    .get(data)
+    .then(response => {
+      logger.info(response);
```

Comment:
Mmm do you need this log ? Also you have to return this response here if you add this